### PR TITLE
Allow PHP 8.4

### PR DESCRIPTION
**Pull request by ruudk**
Needed for https://github.com/moneyphp/money/pull/796

@frederikbosch

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     }
   ],
   "require": {
-    "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
+    "php": "^8.0",
```
Comment (frederikbosch) [inline composer.json]: Same reasoning applies. We keep our strategy.

Comment (ruudk) [reply composer.json]: Changed

File: composer.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     }
   ],
   "require": {
-    "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
+    "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```
Comment (frederikbosch) [inline composer.json]: Sorry, forgot to add, could you drop 8.0? We also do not support versions that do not receive any security updates anymore.

Comment (ruudk) [reply composer.json]: Good idea!

**Comment by frederikbosch** [comment]
Perfect, thanks again

**Pull request merged**